schema/spacewalk/upgrade/susemanager-schema-4.2.7-to-susemanager-schema-4.2.8/701-rhn_server_fqdn.sql

```diff
@@ -0,0 +1,6 @@
+
+ALTER TABLE rhnServerFQDN ADD COLUMN IF NOT EXISTS is_primary VARCHAR(1);
```

Comment:
Would it make sense to set one per server as the primary as part of the migration?